resources/config.applescript

```diff
@@ -3,7 +3,7 @@
 -- configurable options --
 
 -- limit number of songs to improve efficiency
-property resultLimit : 15
+property resultLimit : 90
```

Comment:
@April-June-August I know you made changes for your own testing, but for this PR, I will also need you to revert the result limit to 15 before I can approve.

```suggestion
property resultLimit : 15
```